source/WorkingPractices/planning_your_change.rst

```diff
@@ -34,7 +34,7 @@ General Considerations
 **How complex is your change likely to be?** (e.g. roughly how many subroutines or lines of code do
 you expect to alter or add?) This is an important consideration as the more complex a change is, the
 more time will be required in development, the more code owners will need to approve it and so forth.
-If a change is overly complex, the developer could consider breaking it up into smaller, more
+If a change is overly complex, the developer should consider breaking it up into smaller, more
```

Comment:
smaller more manageable, and where possible, 'self contained' tickets

Note: GitHub is telling me the spelling of "managable" below is incorrect and should be manageable.